packages/providers/src/sdk/CryptKeeperInjectedProvider.ts

```diff
@@ -339,44 +338,45 @@ export class CryptKeeperInjectedProvider {
         merkleProofArtifacts,
         merkleProof,
       },
-      source: "offscreen",
     }) as Promise<SemaphoreProof>;
   }
 
   /**
-   * Generates an RLN proof.
+   * Generates an RLN (Reputation Linked-Note) proof based on the provided RLN proof request.
    *
-   * @param {string} externalNullifier - The external nullifier.
-   * @param {string} signal - The signal.
-   * @param {string | MerkleProofArtifacts} merkleProofArtifactsOrStorageAddress - The merkle proof artifacts or storage address.
-   * @param {string} rlnIdentifier - The RLN identifier.
-   * @returns {Promise<RLNFullProof>} A Promise that resolves to the RLN proof.
+   * @param {string} rlnIdentifier - The RLN identifier as a string.
```

Comment:
I'd use object instead of separate params.

Reply:
I was thinking about this actually but this way is better for allowing developer to deal with this API easily. 
Which I think much easier that showing just an object of inputs. 

![Screenshot_20230801_134606](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/36774944/d4701f4b-85d4-4adc-9032-e5ffafe63424)